style.css

```diff
@@ -113,12 +127,56 @@ li {
   margin: 0 8px;
 }
 
+@media (min-width: 640px) {
```

Comment:
- [ ] You have done an excellent job with your desktop version design :muscle: but the issue with the media query is that for larger devices it's recommended to use a `min-width` value between `992px` and `1100px` (e.g., `1080px`). Please, update this value to the recommended range :wink:

Reply:
Hi @alonsofl
 
You can give a example `code` please! :pray:

Reply:
@hendridg, you just have to change the `640px` to a number between `992px` and `1100px` because that range it's recommended for larger devices :wink:
Examples :pushpin:
```
@media (min-width: 992px) {
```
or
```
@media (min-width: 1080px) {
```